Clean up formatting in tournament articles (#5861)
ppy/osu-wiki: wiki/Tournaments/CIC/2018/en.md

10. Lag is not a valid reason to nullify a map.
11. If there are any problems during the match occurence, the Tournament Management will make a decision based on referee's report.
12. It is expected to respect and be polite to each other. Penalties will be given if participants violates.
    - If participants try to provoke other participants, participants who start the provocation will be reported to the CIC staff and blacklisted in the next instalment of CIC.
    - Usage of any illegal programs will be reported to the osu!staff and will be blacklisted in the next instalment of CIC.

>     - If there is anyone try to play as the participants, their account will be blacklisted in the next instalment of CIC for both participant and the helper.

Reviewer: ```suggestion
    - If there is anyone trying to play in place of any of the participants, their account will be blacklisted in the next instalment of CIC for both participant and the helper.
```

I'm not sure how this wording can be improved
